### PR TITLE
added pretrack events

### PR DESCRIPTION
**Pull request by yarneo**

### CODE REVIEW

**Comment by jayjo** [comment]
@rickydoar Thoughts?

File: lib/report/constants.js [hunk 1]
```diff
@@ -261,6 +261,12 @@ export const EVENTS = {
   $show_survey: `Show Survey`,
   $top_events: `Your Top Events`,
   $signup: `Signup`,
+  $ae_first_open: `First App Open`,
+  $ae_session: `App Session`,
+  $ae_updated: `App Updated`,
+  $ae_crashed: `App Crashed`,
+  $ae_notif_opened: `Notification Opened`,
+  $ae_iap: `In-App Purchase`,
```
Comment (tdumitrescu) [inline lib/report/constants.js]: sorry to be a nitpicker, please keep this list (`EVENTS`) alphabetized

Comment (yarneo) [inline lib/report/constants.js]: done!

**Comment by tdumitrescu** [comment]
published in 0.70.1

**Pull request merged**